src/odometry.c

```diff
@@ -281,19 +282,42 @@ void process_int(
     {
       err->state[id] = value;
       err->time[id] = receive_time;
+
       if (value != ERROR_NONE)
         yprintf(OUTPUT_LV_ERROR, "Error: The driver of motor_id %d returned ", id);
       if (value & ERROR_LOW_VOLTAGE)
+      {
         yprintf(OUTPUT_LV_ERROR, "ERROR_LOW_VOLTAGE ");
+        unknown_err = 0;
+      }
       if (value & ERROR_HALL_SEQ)
+      {
         yprintf(OUTPUT_LV_ERROR, "ERROR_HALL_SEQ ");
+        unknown_err = 0;
+      }
       if (value & ERROR_HALL_ENC)
+      {
         yprintf(OUTPUT_LV_ERROR, "ERROR_HALL_ENC ");
+        unknown_err = 0;
+      }
       if (value & ERROR_WATCHDOG)
+      {
         yprintf(OUTPUT_LV_ERROR, "ERROR_WATCHDOG ");
+        unknown_err = 0;
+      }
+      if (value & ERROR_HEARTBEAT)
+      {
+        yprintf(OUTPUT_LV_ERROR, "ERROR_HEARTBEAT ");
+        unknown_err = 0;
+      }
 
       if (value != ERROR_NONE)
+      {
+        if (unknown_err)
```

Comment:
Unknown error bits can also be set even if known error bits are set.
Clearing known error bits in above if-blocks and checking remaining bits should work.

Reply:
done.